### PR TITLE
Replace '+' symbols with empty spaces

### PR DESCRIPTION
**Pull request by kimlisa**
part of https://github.com/gravitational/teleport/issues/3774

#### Description
- Use native URLSearchParams to handle escape symbols
- Add unit test

#### Screenshot
Screenshot from 2020-06-17 21-47-43

### CODE REVIEW

File: packages/teleport/src/services/history.js [hunk 1]
```diff
@@ -124,7 +124,7 @@ export function getUrlParameter(name, path) {
   for (var i = 0; i < vars.length; i++) {
     var pair = vars[i].split('=');
     if (decodeURIComponent(pair[0]) == name) {
-      return decodeURIComponent(pair[1]);
+      return decodeURIComponent(pair[1]).replace(/\+/g, ' ');
```
Comment (alex-kovoy) [inline packages/teleport/src/services/history.js:127]: decodeURIComponent should correctly decode the text. If it has + then it should preserve it. The question is why is there "+" in the first place?

Comment (kimlisa) [reply packages/teleport/src/services/history.js]: as disucssed in slack, replaced with URLSearchParams

File: packages/teleport/src/services/history.js [hunk 2]
```diff
@@ -118,15 +118,7 @@ const match = url => route => {
...
-  const vars = query.split('&');
-  for (var i = 0; i < vars.length; i++) {
-    var pair = vars[i].split('=');
-    if (decodeURIComponent(pair[0]) == name) {
-      return decodeURIComponent(pair[1]);
-    }
-  }
-
-  return '';
+  const params = new URLSearchParams(path || window.location.search);
```
Comment (alex-kovoy) [inline packages/teleport/src/services/history.js:121]: Lets remove `window.location.search` and good to go!

Comment (kimlisa) [reply packages/teleport/src/services/history.js]: revised

**Pull request merged**